src/builder/possible_value.rs

```diff
@@ -180,21 +218,25 @@ impl PossibleValue {
     #[cfg(feature = "help")]
     pub(crate) fn get_visible_quoted_name(&self) -> Option<std::borrow::Cow<'_, str>> {
         if !self.hide {
-            Some(if self.name.contains(char::is_whitespace) {
-                format!("{:?}", self.name).into()
-            } else {
-                self.name.as_str().into()
-            })
+            Some(quote_if_whitespace(&self.name))
         } else {
             None
         }
     }
 
+    /// Get the names of all visible aliases, but wrapped in quotes if they contain whitespace
+    pub fn get_quoted_visible_aliases(&self) -> Vec<std::borrow::Cow<'_, str>> {
```

Comment:
This feels too specialized to make `pub`

Reply:
@epage `clap_mangen` needs to call it.

Reply:
`clap_mangen` is choosing to format quoted aliases the same way as the `--help`.  I don't see that as a hard requirement to be driving code sharing and rather only couple code together when its driven by requirements

Reply:
@epage What would you suggest instead? I don't want to leave them out of the manpages entirely.

Reply:
Duplicate what you need